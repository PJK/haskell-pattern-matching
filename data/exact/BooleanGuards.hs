module BooleanGuards where

-- TODO add me
