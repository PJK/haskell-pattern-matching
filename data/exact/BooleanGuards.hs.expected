{
    "tag": "AnalysisSuccess",
    "contents": [
        [
            "bguard",
            {
                "tag": "Redundant",
                "contents": [
                    {
                        "tag": "WildcardPattern",
                        "contents": []
                    },
                    {
                        "tag": "GuardPattern",
                        "contents": [
                            {
                                "tag": "ConstructorPattern",
                                "contents": [
                                    "True",
                                    []
                                ]
                            },
                            {
                                "tag": "BExp",
                                "contents": {
                                    "tag": "Otherwise",
                                    "contents": []
                                }
                            }
                        ]
                    }
                ]
            }
        ]
    ]
}
